server/routerlicious/packages/services-utils/src/conversion.ts

```diff
@@ -8,3 +8,36 @@
  */
 export const parseBoolean = (value: any): boolean =>
     typeof value === "boolean" ? value : value === "true";
+
+/**
+ * Converts the given number array into an array of ranges
+ * Example: [1, 2, 3, 4, 5, 6] to [[1, 6]]
+ * [1, 2, 3, 5, 6] to [[1,3],[5,6]]
+ */
+export function convertNumberArrayToRanges(numberArray: number[]): number[][] {
```

Comment:
Since you need this in the Lambdas package (isomorphic), it doesn't rely on any `node` dependencies, and it's only used in Scriptorium, I would strongly suggest just moving it into a `helpers.ts` file under `lambdas/src/utils`. Otherwise, you have to move it to the `services-client` package instead of `services-utils`.